docs/cli-reference.adoc

```diff
@@ -627,12 +627,20 @@ _________________
 
 Modifies odo specific configuration settings within the global preference file.
 
+[NOTE]
+====
+By default the path to the global preference file is `/home/username/.odo/preferece.yaml`, it is stored in the `GLOBALODOCONFIG` variable. You can set up a custom path by changing the value of the variable `GLOBALODOCONFIG="/new_path/preference.yaml"`.
```

Comment:
```/home/username/.odo/preferece.yaml``` looks platform specific. Can it be something more generic like ```~/.odo/preferece.yaml```

```it is stored in the `GLOBALODOCONFIG` variable``` -> it is stored in the environment variable `GLOBALODOCONFIG`

You can set up a custom path by changing the value of the variable `GLOBALODOCONFIG="/new_path/preference.yaml"` -> You can set up a custom path by updating the value of the environment variable to  a new preference path, for example `GLOBALODOCONFIG="new_path/preference.yaml"`

I would recommend @mohammedzee1000 to comment here